src/js/components/Button/__tests__/Button-test.js

```diff
@@ -220,3 +220,13 @@ test('Button is clickable', () => {
   button[0].props.onClick();
   expect(onClick).toBeCalled();
 });
+
+test('renders tag', () => {
+  const component = renderer.create(
+    <Grommet>
+      <Button tag="span" />
```

Comment:
I'm sorry i did not mention this. but can we use `as` to align with styled-component and the other anchor pr ?

Reply:
Yes i am on it. Should i check other components as well? At least the Text component is using tag https://github.com/grommet/grommet/blob/master/src/js/components/Text/index.d.ts#L10